src/main/java/com/echobox/api/linkedin/types/urn/URN.java

```diff
@@ -58,6 +58,16 @@ public URN(String entityType, String id) {
     this.entityType = entityType;
     this.id = id;
   }
+  
+  /**
+   * Instantiates a new Urn.
+   *
+   * @param entityType the entity type
+   * @param id the id
+   */
+  public URN(URNEntityType entityType, String id) {
+    this(entityType.name(), id);
```

Comment:
I think it needs to be lower case...

```suggestion
    this(entityType.name().toLowerCase(), id);
```